packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -0,0 +1,36 @@
+import { h } from '@stencil/core';
+import { getNodeName } from '../get-node-name.function';
+
+import { OzonContentNode } from '../ozon-content-node.interface';
+
+export class OzonContentFiguurNode implements OzonContentNode {
+  name = [
+    'Figuur',
+  ];
+
+  render(node: Element) {
+    const childNodes = Array.from(node.childNodes);
+    const titel = childNodes.find(n => getNodeName(n) === 'Titel')?.textContent;
+    const bijschrift = childNodes.find(n => getNodeName(n) === 'Bijschrift')?.textContent;
```

Comment:
Volgens de STOP specificatie https://koop.gitlab.io/STOP/standaard/1.3.0/tekst_xsd_Element_tekst_Bijschrift.html#Bijschrift kan er complexe inhoud in `Bijschrift`.

Als ik naar de type inhoud kijk, lijkt het best veel op `Al`, misschien dat we Bijschrift gelijk kunnen trekken? En anders de kindjes van Bijschrift doormappen?

Reply:
#1730